.tekton/build-dm-verity-image-task.yaml

```diff
@@ -0,0 +1,395 @@
+        rsync -ra "/var/workdir/source" "$SSH_HOST:$BUILD_DIR/"
+
+        # this unquoted heredoc allows expansions for the image name
+        cat >scripts/script-build.sh <<REMOTESSHEOF
+        #!/bin/sh
+        set -e
+
+        export BUILD_DIR="$BUILD_DIR"
+        export OUTPUT_IMAGE="$OUTPUT_IMAGE"
+        export BUILDAH_IMAGE="$BUILDAH_IMAGE"
+
+        REMOTESSHEOF
+
+        # this quoted heredoc prevents expansions and command substitutions. the env vars are evaluated on the remote vm
+        cat >>scripts/script-build.sh <<'REMOTESSHEOF'
+
+        mkdir -p output/qcow2/
+        echo "RUNNING BUILD"
+
+        time sudo podman build --authfile=$BUILD_DIR/.docker/config.json -v $BUILD_DIR/entitlement:/etc/pki/entitlement:Z -t builder -f $BUILD_DIR/source/konflux/Dockerfile $BUILD_DIR/source
+
+        time sudo podman run -d --authfile=$BUILD_DIR/.docker/config.json --rm -it --privileged --pull=newer \
+          -v ${BUILD_DIR}:/workspace \
+          -v $(pwd)/output:/output \
+          -v $BUILD_DIR/scripts/script-verity.sh:/script-verity.sh \
+          -v /lib/modules:/lib/modules:ro,Z \
+          --user 0 \
+          --security-opt=apparmor=unconfined \
+          --security-opt=seccomp=unconfined \
+          --mount type=bind,source=/dev,target=/dev \
+          --mount type=bind,source=/run/udev,target=/run/udev \
+          --env=LIBGUESTFS_BACKEND=direct \
+          builder
+
+        time sudo podman exec --latest bash -c 'until virsh list --all >/dev/null 2>&1; do sleep 2; done'
+
+        time sudo podman exec -t --latest virt-install --virt-type qemu --cpu host-model --os-variant rhel9.0 --arch x86_64 --boot uefi --name disk --memory 8192 --location /workspace/rhel.iso --disk path=/output/qcow2/disk.qcow2,format=qcow2,bus=scsi,size=3 --initrd-inject=/workspace/source/helpers/rhel9-dm-root.ks --nographics --extra-args 'console=ttyS0 inst.ks=file:/rhel9-dm-root.ks' --transient
+
+        time sudo podman exec -t --latest /script-verity.sh
+
+        time sudo podman run --authfile=$BUILD_DIR/.docker/config.json --rm -it --privileged --pull=newer --security-opt label=type:unconfined_t \
+          -v $BUILD_DIR/.docker:/.docker \
+          -v $(pwd)/output:/output \
+          -v /var/lib/containers/storage:/var/lib/containers/storage \
+          -v $BUILD_DIR/scripts/script-push.sh:/script-push.sh \
+          -v $BUILD_DIR/tekton-results:/tekton-results \
+          "$BUILDAH_IMAGE" \
+          /script-push.sh
+
+        REMOTESSHEOF
+
+        cat >>scripts/script-verity.sh <<'REMOTESSHEOF'
+        #!/bin/bash
+        set -ex
+
+        ADDON_SBAT="sbat,1,SBAT Version,sbat,1,https://github.com/rhboot/shim/blob/main/SBAT.md
+        coco-podvm-uki-addon,1,Red Hat,coco-podvm-uki-addon,1,mailto:secalert@redhat.com"
+        
+        DISK=/output/qcow2/disk.qcow2
+        NBD_DEVICE=/dev/nbd0
+        EFI_PN=nbd0p1
+        ROOT_PN=nbd0p2
+
+        LUKS_MINIMAL_SPACE_MB=100
```

Comment:
changed to 2500
https://github.com/confidential-devhub/coco-podvm-scripts/commit/bca9568af24d65fcd35c3480f2cd693f59c37917